src/threading.ts

```diff
@@ -247,11 +270,18 @@ export class BytecodeStackFrame implements IStackFrame {
 
     // Run until we get the signal to return to the thread loop.
     while (!this.returnToThreadLoop) {
-      var op = code.readUInt8(this.pc);
-      if (!RELEASE && logging.log_level === logging.VTRACE) {
-        vtrace(`  ${this.pc} ${annotateOpcode(op, this, code, this.pc)}`);
+      var op = method.getOp(this.pc, code);
```

Comment:
Oh I see; I haven't seen any JITs before. I have coded the changes for counter-increments-during-jumps and will push tomorrow. Although I don't see any significant performance improvements, we might be able to tune it further post-merge.